web/src/message_edit.ts

```diff
@@ -768,11 +826,12 @@ export function start_inline_topic_edit($recipient_row) {
     $(".topic_edit_spinner").hide();
     const msg_id = rows.id_for_recipient_row($recipient_row);
     const message = message_lists.current.get(msg_id);
+    assert(message?.type === "stream");
     let topic = message.topic;
     if (topic === compose_state.empty_topic_placeholder()) {
         topic = "";
     }
-    const $inline_topic_edit_input = $form.find(".inline_topic_edit");
+    const $inline_topic_edit_input = $form.find<HTMLInputElement>(".inline_topic_edit");
```

Comment:
Adjust the selector string to guarantee type safety.

```suggestion
    const $inline_topic_edit_input = $form.find<HTMLInputElement>("input.inline_topic_edit");
```